R/install-remote.R

```diff
@@ -32,11 +32,23 @@ install_remotes <- function(remotes, ...) {
 # Add metadata
 add_metadata <- function(pkg_path, meta) {
   path <- file.path(pkg_path, "DESCRIPTION")
-  desc <- read_dcf(path)
+  if (file.exists(path)) {
+    desc <- read_dcf(path)
+
+    desc <- modifyList(desc, meta)
+
+    write_dcf(path, desc)
+  }
 
-  desc <- modifyList(desc, meta)
+  path <- file.path(pkg_path, "Meta", "package.rds")
+  if (file.exists(path)) {
```

Comment:
Yes, some functions will look to one or the other, so both files (if present) need to be modified. It might be a good idea to add a warning that neither file exists and metadata couldn't be added?

```
if (file.exists(source_desc)) {}

if (file.exists(binary_desc)) {}

if (!file.exists(source_desc) && !file.exists(binary_desc)) {
    warning("No DESCRIPTION found, metadata not added!", call. = FALSE)
}
```